src/common/Makefile.am

```diff
@@ -94,12 +89,23 @@ libcommon_internal_la_SOURCES += \
         common/aix_errno.cc
 endif
 
-if WITH_RBD
```

Comment:
common/blkdev.cc is used by FileJournal.cc, nothing do with rbd. this line was introduced by solaris port RP, it should be wrong.